api_tests/lib_sdk/actors/actor.ts

```diff
@@ -637,18 +660,31 @@ export class Actor {
     }
 
     private async initializeDependencies() {
+        let lightning = false;
         for (const ledgerName of [
             this.alphaLedger.name,
             this.betaLedger.name,
         ]) {
-            await this.wallets.initializeForLedger(ledgerName);
+            if (ledgerName === "lightning") {
+                lightning = true;
+            }
+            await this.wallets.initializeForLedger(
+                ledgerName,
+                this.logger,
+                this.logRoot,
+                this.config
+            );
         }
 
-        this.comitClient = new ComitClient(
-            this.wallets.getWalletForLedger("bitcoin").inner,
-            this.wallets.getWalletForLedger("ethereum").inner,
-            this.cnd
-        );
+        // Once `ComitClient` can be built with lightning, this hack should
+        // be removed and ComitClient built with the wallets available
+        if (!lightning) {
```

Comment:
Ditch the mess with the mutable boolean var and just check it here:

```suggestion
        if (this.alphaLedger.name != "lightning" && this.betaLedger.name != "lightning") {
```